sdk/keyvault/keyvault-secrets/src/lro/recover/operation.ts

```diff
@@ -110,6 +110,8 @@ export class RecoverDeletedSecretPollOperation extends KeyVaultSecretPollOperati
         } else if (error.statusCode !== 404) {
```

Comment:
This reads as:

403 - ignored
404 - throw error
anything else - store the error but do not throw.

Is that what you intended?

Reply:
oh thank you! my bad